addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -324,6 +324,9 @@
                         </div>
                         <field name="name" attrs="{'readonly': [('state', 'not in', ('draft', 'confirm'))]}" widget="text"/>
                         <field name="user_id" invisible="1"/>
+                        <field name="leave_type_support_document" invisible="1"/>
+                        <label for="support_documents_ids" string="Attach Supporting Documents" attrs="{'invisible': ['|', ('leave_type_support_document', '=', False), ('state', 'not in', ('draft', 'confirm', 'validate1'))]}"/>
+                        <field name="support_documents_ids" widget="many2many_binary" attrs="{'invisible': ['|', ('leave_type_support_document', '=', False), ('state', 'not in', ('draft', 'confirm', 'validate1'))]}" nolabel="1"/>
```

Comment:
I believe the field should be readonly instead of invisible, no?
I think we should still be able to view uploaded documents when the leave is confirmed.

Reply:
@kba-odoo , We already have those attachment in bottom chatter so once the leave is confirmed user can check those attachment from chatter attachment box. that's why we make them invisible.

Thanks